.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "airbnb",
+  "extends": ["airbnb", "prettier"],
```

Comment:
is this not already merged??

Reply:
@LucaMele I guess this is still in my "develop-v2-raphael" branch.

Reply:
is weird because on `develop-v2` this change is already done so it should not show up here as a diff... im confused

Reply:
True. We merged it in a separate PR into develop-v2, so we can go with this as it does not change anything.